server/models/airplaneSchema.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const airplaneSchema = new mongoose.Schema(
+  {
+    company: {
```

Comment:
And how much seats there should be? from 1A to ?

Reply:
i thought if it will be good to add these objects in airplane shcema and not create seperate SeatSchema, like in airplane i will have -
 seat:{
    [{seat: 1A, available: true(or false if its not available), class: busness or econom or standart}]
}